docker/README.md

```diff
@@ -0,0 +1,241 @@
+# Athenz on Docker
+
+<a id="markdown-index" name="index"></a>
+## Index
+<!-- TOC depthFrom:2 updateOnSave:true -->
+
+- [Athenz on Docker](#Athenz-on-Docker)
+    - [Index](#Index)
+    - [Prerequisite](#Prerequisite)
+    - [Build Athenz](#Build-Athenz)
+    - [Deploy Athenz](#Deploy-Athenz)
+    - [Cleanup](#Cleanup)
+    - [Configuration Details](#Configuration-Details)
+    - [Useful Commands](#Useful-Commands)
+    - [[WIP] deploy with docker-stack](#WIP-deploy-with-docker-stack)
+    - [TO-DO](#TO-DO)
+    - [Important Files](#Important-Files)
+
+<!-- /TOC -->
+
+<a id="markdown-prerequisite" name="prerequisite"></a>
+## Prerequisite
+- docker
+- make
+
+<a id="markdown-build-athenz" name="build-athenz"></a>
+## Build Athenz
+
+```bash
+cd `git rev-parse --show-toplevel`/docker
+
+# there are a lot of build logs. You may want to check it inside a log file later on.
+make build | tee ./athenz-docker-build.log
+```
+
+<a id="markdown-deploy-athenz" name="deploy-athenz"></a>
+## Deploy Athenz
+
+- development evnvironment
+    - deploy commands
+        ```bash
+        cd `git rev-parse --show-toplevel`/docker
+
+        # 1. set passwords (P.S. values in *.properties files will overwrite these values)
+        source ./setup-scripts/0.export-default-passwords.sh
+
+        # 2. generate key-pairs, certificates and keystore/truststore
+        make setup-dev-config
+
+        # 3. (once ONLY) create docker network
+        make setup-docker-network
+
+        # 4.1 (optional) if you are running web browser and docker containers in the same host
+        export HOSTNAME=localhost
+        # 4.2. run Athenz
+        make run-docker-dev
+        ```
+    - Note for UI
+        1. As the self-signed certificates are not trusted by the browser and do not have the `${HOSTNAME}` in the SAN field, you need to ignore the browsers warning message explicitly for UI to work.
+            - ZMS certificate: access `echo https://${HOSTNAME}:${ZMS_PORT:-4443}/zms/v1/status`
```

Comment:
@WindzCUHK 
You can mention something like:
Instead of writing ZMS Certificate, you can mention something like:
To ignore the browser warning for ZMS certificate, access the following URL 
echo https://${HOSTNAME}:${ZMS_PORT:-4443}/zms/v1/status

and the same for UI Certificate as well

Reply:
@chandrasekhar1996 

Thank you for the help.
I updated the session as below.
https://github.com/yahoo/athenz/blob/c102fcf958cda4c595a31947d394db1a8ec62bda/docker/README.md#deploy-athenz